pcmanfm/preferencesdialog.cpp

```diff
@@ -80,15 +80,15 @@ void PreferencesDialog::initIconThemes(Settings& settings) {
         // load xdg icon themes and select the current one
         QHash<QString, QString> iconThemes;
         // user customed icon themes
-        findIconThemesInDir(iconThemes, QString(g_get_home_dir()) % "/.icons");
+        findIconThemesInDir(iconThemes, QFile::decodeName(g_get_home_dir()) + QStringLiteral("/.icons"));
```

Comment:
Thank you very much for the hard work!

Isn't `QString::fromUtf8` better here? We haven't used `QFile::decodeName` anywhere else — in pcmanfm-qt or libfm-qt.

Reply:
Actually, `QFile::decodeName` is (in Qt → `qfile.h`):
```c++
QString::fromUtf8(NAME).normalized(QString::NormalizationForm_C);
```
The `normalized` part should have been already done in `g_get_home_dir`.